mypyc/ir/ops.py

```diff
@@ -1305,6 +1309,8 @@ def sources(self) -> List[Value]:
         return [self.lhs, self.rhs]
 
     def to_str(self, env: Environment) -> str:
+        if self.lhs is self.rhs and self.op == BinaryIntOp.NOR:
+            return env.format('%r = !%r', self, self.lhs)
```

Comment:
This is confusing, since `!r` in C means a different thing than NOR. `~r` would match the semantics, but it would seem unexpected to me.